Userland/Applications/FileManager/main.cpp

```diff
@@ -93,7 +93,7 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
 
     TRY(Core::System::pledge("stdio thread recvfd sendfd cpath rpath wpath fattr proc exec unix"));
 
-    Config::pledge_domains({ "FileManager", "WindowManager" });
+    Config::pledge_domains({ "FileManager", "WindowManager", "Maps" });
```

Comment:
Does this override maps position and zoom data? If so, we should probably have an option for the widget to not store its config back.